apps/public-docsite-resources/config/api-docs.js

```diff
@@ -108,6 +108,7 @@ module.exports = {
       'MultiStackedBarChart',
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33369&commitId=e6df356a163b303dd38ee31308284479b212d844&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|8| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2213| Changed|
|Drawer.overlay drawer full.chromium.png|1150| Changed|
|Drawer.Full Overlay RTL.chromium.png|1168| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>